### ISSUE TITLE
Problem with new version of crystal?, openssl not found.

### ISSUE THREAD

**Issue by raydf** [will/crystal-pg#50]
```bash
require "pg"
^

in ./libs/pg/pg.cr:1: while requiring "./pg/*"

require "./pg/*"
^

in ./libs/pg/pg/connection.cr:1: while requiring "../pq/*"

require "../pq/*"
^


in ./libs/pg/pq/connection.cr:6: while requiring "openssl/openssl": can't find file 'openssl/openssl' relative to '/home/test/libs/pg/pq'

require "openssl/openssl"
^
```

**Comment by raydf** [comment]
OK, as a workaround i changed to branch crystal018, sorry i didn't read the pull request history. Again, thanks for your amazing work in this library.

**Comment by will** [comment]
Released 0.8.0, thanks for report even though it was being worked on :)
